CppCoreGuidelines.md

```diff
@@ -3378,7 +3378,7 @@ The standard library requires that all classes it deals with have destructors th
 None have succeeded to come up with a general scheme.
 This can be be a real practical problem: For example, what about a sockets that won't close?
 The writer of a destructor does not know why the destructor is called and cannot "refuse to act" by throwing an exception.
-See <a =href="#Sd-dtor">discussion</a>.
+See [discussion](#Sd dtor).
```

Comment:
Duh, it undoes a fix I provided myself. I assume you talk about the dash? Fixed that one. and pushed again.